README.md

```diff
@@ -1,5 +1,41 @@
-A MongoDB Dialect for the Hibernate ORM
+# A MongoDB Dialect for the Hibernate ORM
+
+This project aims to provide a library to seamlessly integrate MongoDB with Hibernate ORM. Hibernate _ORM_ is a powerful **O**bject-**r**elational **m**apping tool. Due to the SQL and JDBC standards, Hibernate ORM could centralize each SQL vendor's idiosyncrasies in the so-called _Hibernate Dialect_. But this project will include a NoSQL member in the Hibernate's Dialect family.
```

Comment:
Let's not refer to MongoDB as NoSQL. We don't use that terminology anymore in any of our documentation/marketing.  IIRC we usually call it a document database.

Reply:
I changed it to "document database"